wordpress/wp-content/plugins/gc-post-meta/resources/js/sidebar.js

```diff
@@ -19,6 +19,7 @@ registerPlugin('gc-post-meta', {
 					{__("GC Post Meta", "gc-post-meta")}
 				</PluginSidebarMoreMenuItem>
 				<PluginSidebar name="gc-post-meta" title={__("Custom Fields", "gc-post-meta")} icon="insert">
+					<GCPostMetaSlotFill.Slot />
 					{type === "post" && <PostFields />}
 					{type === "job" && <JobFields />}
 					{type === "team" && <TeamMemberFields />}
```

Comment:
Do we still need these ones?

Reply:
For now until I talk to Omar and move over them over --- the PR bot is looking for them in single fields at the moment.